src/main/java/org/spongepowered/mod/mixin/server/MixinServerConfigManager.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.spongepowered.mod.mixin.server;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.management.ServerConfigurationManager;
+import net.minecraft.util.ChatComponentTranslation;
+import net.minecraft.util.IChatComponent;
+import net.minecraftforge.fml.common.FMLCommonHandler;
+import net.minecraftforge.fml.common.gameevent.PlayerEvent;
+import org.spongepowered.api.event.entity.living.player.PlayerJoinEvent;
+import org.spongepowered.api.text.message.Message.Translatable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.mod.text.message.SpongeMessage;
+import org.spongepowered.mod.text.message.SpongeMessageTranslatable.SpongeMessageTranslatableBuilder;
+
+@Mixin(ServerConfigurationManager.class)
+public class MixinServerConfigManager {
+
+    private ChatComponentTranslation tmpJoinMessage;
+
+    // TODO Fix initializeConnectionToPlayer injection at runtime - it doesn't
+    // seem to know about FML's patches
```

Comment:
Could someone take a look at this please? The [build fails](https://travis-ci.org/SpongePowered/Sponge/jobs/51739641#L522-L527) because it can't find the mappings for `initializeConnectionToPlayer`.
From some investigation the mappings don't seem to take into account FML's patch, [ServerConfigurationManager.java.patch](https://github.com/MinecraftForge/FML/blob/master/patches/minecraft/net/minecraft/server/management/ServerConfigurationManager.java.patch#L8)